tensorrt/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tensorrt-rs"
-version = "0.3.0"
+version = "1.0.0"
```

Comment:
Same thing here as on the tensorrt-sys library. Version should be 0.4.0 instead of 1.0 since we don't have a stable API as of yet.